compliance/portugal.mdx

```diff
@@ -0,0 +1,157 @@
+
+This article provides comprehensive guidance on [ACTUD/QR invoicing requirements](#actudqr-b2b-b2g), [SAF-T reporting](#saft-pt-b2b-b2g), and [Portuguese invoicing regulations](#regulation).
+
+Invopop provides comprehensive coverage for Portuguese invoicing through dedicated apps for AT Portugal and InvoiceXpress Portugal, as well as integrations with platforms like Stripe and Chargebee. This allows businesses to issue compliant ACTUD invoices efficiently and manage white-label (B2B2B) operations.
+
+## Portugal related resources
+
+|    |    |
+|-----------|------------|
+|  Compliance |  🇧🇪&nbsp;&nbsp;Portugal compliance (you're here) <br /> <Icon icon="timeline" /> [Compliance timeline](/timelines/portugal)|
+|  Apps |  <Icon icon="https://assets.invopop.com/apps/at-pt/icon.svg" /> [AT Portugal](/apps/at-portugal) |
+|  Guides | <Icon icon="book" /> [AT Portugal Guide](/guides/at-pt) |
+|  GOBL | <Icon icon="https://assets.invopop.com/icons/gobl.svg" />  [Portugal Tax Regime](https://docs.gobl.org/regimes/pt)<br /> <Icon icon="https://assets.invopop.com/icons/gobl.svg" /> [Portugal SAF-T Addon ](https://docs.gobl.org/addons/pt-saft-v1) |
+
+
+## Invoicing in Portugal
+
+Portugal does not mandate a specific electronic invoice format. Instead, businesses report tax information to the Portuguese Tax Authority (AT - Autoridade Tributária e Aduaneira) either periodically through SAF-T (PT) reports or in real time via web service.
+
+<AccordionGroup>
+  <Accordion title="ACTUD/QR (B2B, B2G)">
+
+Although no format is required, all electronic invoices must include ATCUD and a QR code.
+
+1. **ATCUD (Unique Code)**: 
+   - Businesses must register their invoice numbering series with the AT
+   - The AT issues a validation code for each registered series
+   - This code must appear on each invoice
+
+2. **QR Code**:
+   - All e-invoices must contain a QR code
+   - The QR code links to the AT's online verification service
+
+
+  |            |             |
+  |------------|-------------|
+  | **Models** | B2B |
+  | **Format** | EN 16931 with CUIS extensions |
+  | **Infrastructure** | Hybrid: public and private platforms |
+  | **Model** | Centralized |
+  | **Scope & Deadline** | All B2B invoices from **Jan 1, 2026**; mandatory for suppliers to public authorities. Optional for B2C simplified invoices. |
+  | **Agency** | [AT Portugal](https://www.portaldasfinancas.gov.pt) |
+  | **Invopop Support** | [AT Portugal App](/apps/at-pt)<br />[InvoiceXpress Portugal App](/apps/pt-invoicexpress) |
+
+
+    <Columns cols="2">
+      <Card title="AT Portugal" size="20" icon="https://assets.invopop.com/apps/at-pt/icon.svg" href="/apps/at-pt" horizontal>Get covered →</Card>
+      <Card title="InvoiceXpress Portugal" icon="https://assets.invopop.com/apps/invoicexpress/icon.svg" href="/apps/pt-invoicexpress" horizontal>Get covered →</Card>
```

Comment:
As mentioned before, I'd double-check with Sam and Juan whether we want to keep advertising InvoiceXpress.